docs/README.md

```diff
@@ -0,0 +1,109 @@
+## Alfresco Identity Service
+
+![Completeness Badge](https://img.shields.io/badge/Document_Level-In_Progress-yellow.svg?style=flat-square)
+
+![Version Badge](https://img.shields.io/badge/Version-Future-blue.svg?style=flat-square)
+
+### Purpose
+The Alfresco Identity Service was initially introduced to the architecture as a means for Client Applications to implement single sign-on
+across components of the Digital Business Platform (via [OpenID Connect](https://openid.net/connect/)) while providing the option to 
+delegate authentication of the user to an external Identity Provider over protocols like SAML, OpenID Connect, and LDAP.
+
+Aspects of Common Identity and Governance are likely to be the responsibility of the Identity Service in the future.
+
+*** 
+
+### Overview
+The Identity Service leverages the open source identity and access management project [Keycloak](https://www.keycloak.org/).  An example
+realm file containing a pre-defined 'alfresco' client and login theme are imported on bootstrap.
+
+Client Applications can obtain a JSON Web Token (JWT) via OpenID Connect which can then be presented in a Bearer Authorization header
+in an HTTP request against REST APIs of Alfresco components such as Process Services and Content Services.
+
+The primary deployment mechanism is via a Kubernetes Helm chart, and typically as part of the [alfresco-infrastructure chart](https://github.com/Alfresco/alfresco-infrastructure-deployment).
+
+*** 
+
+### Artifacts and Guidance
+
+* Source Code: https://github.com/Alfresco/alfresco-identity-service
```

Comment:
Should we also list the themes' repository here?

Reply:
Indeed, updated.